File: code/model/group.py

from sparrow import *

from .owentity import *
from .wall import Wall

class Group(RTOwEntity):
    key = GID = KeyProperty()
    title = Property(str)
    description = Property(str)
    public = Property(bool)
    wall = Reference(Wall)  # Not RT because it never really changes

File: code/js/prototypes/group.js
Group.prototype = new DataType();
Group.prototype.constructor = Group;
Group.prototype._key = ["GID"];

function Group(GID, title, description, public, wall_WID) {
	this.GID = GID;
	this.title = title;
	this.description = description;
    this.public = public;
	this.wall_WID = wall_WID;
}
